CMakeLists.txt

```diff
@@ -141,7 +141,7 @@ check_ipo_supported(RESULT SCP_IPO_SUPPORTED)
 cmake_dependent_option(
     SCP_ENABLE_IPO
     "Enable the interprocedural optimization (IPO) if supported?"
-    "${SCP_ENABLE_IPO_INIT}" "DEFINED SCP_ENABLE_IPO_INIT" "${SCP_ENABLE_IPO}")
+    "${SCP_ENABLE_IPO_INIT}" "DEFINED SCP_ENABLE_IPO_INIT" TRUE)
```

Comment:
We don't intend to enable IPO by default, hence, it's being enabled/disabled by setting SCP_ENABLE_IPO_INIT in product specific Firmware.cmake file. From what I understand, this change causes IPO to be enabled by default when SCP_ENABLE_IPO_INIT is not defined. 
Some platforms do not behave as intended when IPO is enabled, so this shouldn't be the case.

Reply:
yes, That's a leftover while testing the change below which should have been removed